frontend/src/scenes/SignIn/SignIn.tsx

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import { Formik, Form, useField } from 'formik';
+import * as Yup from 'yup';
+import './SignUp.sass';
+import { login } from '../../services/authService';
+import { setLocalStorageItem } from '../../helpers/localStorageHelper';
+
+const InputField = ({ label, ...props }: { label: string, name: string, type: string, placeholder?: string }) => {
+  const [field, meta] = useField(props);
+  return (
+    <div className="inputField form-group d-flex flex-column">
+      <span>{label}</span>
+      <input className={`text-input ${meta.touched && meta.error ? 'invalid-input' : ''}`} {...field} {...props} />
+      {meta.touched && meta.error ? (
+        <div className="error">{meta.error}</div>
+      ) : null}
+    </div>
+  );
+};
+
+export const SignUp = (props: any) => {
+  return (
+    <div className="signUp">
+      <h1 className="text-center p-5">Sign in</h1>
+      <Formik
+        initialValues={{
+          email: '',
+          password: ''
+        }}
+        validationSchema={Yup.object().shape({
```

Comment:
You can define validationSchema in separate file or in the top of this file. 
In such way defenition happens just once, not on every render of SignUp component